bftengine/src/bftengine/ReplicasInfo.cpp

```diff
@@ -112,7 +112,7 @@ ReplicasInfo::ReplicasInfo(const ReplicaConfig& config,
         for (auto i = start; i < (end - ((uint16_t)config.operatorEnabled_)); ++i) {
           ret.insert(i);
         }
-        ret.insert(end + config.numOfClientServices - 1);
+        if (config.operatorEnabled_) ret.insert(end + config.numOfClientServices - 1);
```

Comment:
Please remove log
if (config.operatorEnabled_) LOG_INFO(GL, "Operator id: " << end);
from _idsOfClientServices{[&config]()
to not be printed twice.